pandas/tests/groupby/test_groupby.py

```diff
@@ -1861,6 +1861,49 @@ def get_result():
                         get_result()
 
                     return
+    else:
```

Comment:
this test is gigantic. umm can you split up (may not be easy).

Reply:
addressing this is a follow-on, yah